### ISSUE TITLE
Concerns about scalability

### ISSUE THREAD

**Issue by jlouis** [mbbx6spp/bash-http-server#2]
Hi Susan,

Is this bash-http-server proper webscale? Will it handle tens of thousands of connections, non-blocking in the style of `Node.js`. And what framework do you propose I'm going to use? Furthermore, what does your stack look like? Are you proper MongoDB webscale here, or did you pick something slow like Postgres?

**Comment by polynomial** [comment]
I agree with @jlouis. But I also want to know, when I deploy this I will no longer be able to ride my fixie because my service is down due to me only deploying it on one node?

**Comment by cransom** [comment]
I found that this did not scale well. But, after porting it to dash (to be closer to the metal and minimize memory requirements), it only crashes a few times a day and is much improved.

**Comment by vjanelle** [comment]
According to #1, this is "not webscale".

**Comment by PeterFCaswell** [comment]
I find that when I hide it behind an Apple service, it can handle as many requests as I throw at it.

**Comment by trane** [comment]
Actually, I've done some real-world benchmarking with apache bench and can say definitively that this is webscale. It only took 10ms per request and handled up to 4 concurrent requests per second without crashing. My jruby app doesn't even start up that fast! And that's on the _new_ jruby 9000!!!!1

So, haters 'gonna h8.

P.S.
Can we ship this in a self executing `.tar` file?

**Comment by mbbx6spp** [comment]
Where are the like/love buttons? <3

**Comment by jlouis** [comment]
Hello.

Is there any update on this issue?
